pkg/appconsts/versioned_consts.go

```diff
@@ -1,100 +1,34 @@
 package appconsts
 
 import (
-	"strconv"
 	"time"
 
-	v1 "github.com/celestiaorg/celestia-app/v3/pkg/appconsts/v1"
-	v2 "github.com/celestiaorg/celestia-app/v3/pkg/appconsts/v2"
-	v3 "github.com/celestiaorg/celestia-app/v3/pkg/appconsts/v3"
+	v4 "github.com/celestiaorg/celestia-app/v4/pkg/appconsts/v4"
 )
 
 const (
-	LatestVersion = v3.Version
+	LatestVersion = v4.Version
 )
 
-// SubtreeRootThreshold works as a target upper bound for the number of subtree
-// roots in the share commitment. If a blob contains more shares than this
-// number, then the height of the subtree roots will increase by one so that the
-// number of subtree roots in the share commitment decreases by a factor of two.
-// This step is repeated until the number of subtree roots is less than the
-// SubtreeRootThreshold.
-//
-// The rationale for this value is described in more detail in ADR-013.
-func SubtreeRootThreshold(_ uint64) int {
-	return v3.SubtreeRootThreshold
-}
-
-// SquareSizeUpperBound imposes an upper bound on the max effective square size.
-func SquareSizeUpperBound(_ uint64) int {
-	if OverrideSquareSizeUpperBoundStr != "" {
-		parsedValue, err := strconv.Atoi(OverrideSquareSizeUpperBoundStr)
-		if err != nil {
-			panic("Invalid OverrideSquareSizeUpperBoundStr value")
-		}
-		return parsedValue
-	}
-	return v3.SquareSizeUpperBound
-}
-
-func TxSizeCostPerByte(_ uint64) uint64 {
-	return v3.TxSizeCostPerByte
-}
-
-func GasPerBlobByte(_ uint64) uint32 {
-	return v3.GasPerBlobByte
-}
-
-func MaxTxSize(_ uint64) int {
-	return v3.MaxTxSize
-}
-
 var (
-	DefaultSubtreeRootThreshold = SubtreeRootThreshold(LatestVersion)
-	DefaultSquareSizeUpperBound = SquareSizeUpperBound(LatestVersion)
-	DefaultTxSizeCostPerByte    = TxSizeCostPerByte(LatestVersion)
-	DefaultGasPerBlobByte       = GasPerBlobByte(LatestVersion)
+	DefaultSquareSizeUpperBound = v4.SquareSizeUpperBound
+	DefaultTxSizeCostPerByte    = v4.TxSizeCostPerByte
+	DefaultGasPerBlobByte       = v4.GasPerBlobByte
+	DefaultVersion              = v4.Version
+	DefaultTimeoutCommit        = v4.TimeoutCommit
+	DefaultUpgradeHeightDelay   = v4.UpgradeHeightDelay
+	DefaultMaxTxSize            = v4.MaxTxSize
+	SubtreeRootThreshold        = v4.SubtreeRootThreshold
 )
 
-func GetTimeoutPropose(v uint64) time.Duration {
-	switch v {
-	case v1.Version:
-		return v1.TimeoutPropose
-	case v2.Version:
-		return v2.TimeoutPropose
-	default:
-		return v3.TimeoutPropose
-	}
-}
-
-func GetTimeoutCommit(v uint64) time.Duration {
-	switch v {
-	case v1.Version:
-		return v1.TimeoutCommit
-	case v2.Version:
-		return v2.TimeoutCommit
-	default:
-		return v3.TimeoutCommit
-	}
+func GetTimeoutCommit(_ uint64) time.Duration {
+	return v4.TimeoutCommit // TODO: remove this fn currently just used in tests, those tests should fail with this currently logic.
```

Comment:
The TODO confuses me because incorrect grammar. Maybe it's no longer relevant?

```suggestion
	return v4.TimeoutCommit
```

Reply:
Added a comment to https://github.com/celestiaorg/celestia-app/issues/4508#issuecomment-2778476693 to mention that this can also be cleaned out